examples/custom_deserialization.rs

```diff
@@ -1,7 +1,7 @@
 use anyhow::Result;
 use scylla::client::session::Session;
 use scylla::client::session_builder::SessionBuilder;
-use scylla::deserialize::DeserializeValue;
```

Comment:
:pick: typo (`Remve`) in commit `Remve DeserializeValue/Row re-exports in deserialize/mod.rs `.